libraries/provider_hollandbackup_mysqldump.rb

```diff
@@ -0,0 +1,26 @@
+require 'chef/provider/lwrp_base'
+require_relative 'helpers'
+
+class Chef
+  class Provider
+    class HollandbackupMysqldump < Chef::Provider::LWRPBase
+      include Chef::DSL::IncludeRecipe
+
+      use_inline_resources if defined?(use_inline_resources)
+
+      action :configure do
+        template 'holland.conf' do
```

Comment:
This appears to be the same template for both 'main' and 'mysqldump' -- could these just be one provider with an attribute for a toggle? It would reduce the duplicated files down to a single attribute (much shorter).